### ISSUE TITLE
Apple has purchased Dark Sky and is shutting down the API

### ISSUE THREAD

**Issue by tinohager** [worldturtlemedia/darksky-api#385]
Apple has purchased Dark Sky and is shutting down the API.
The API will continue to function through the end of 2021.
[More Informations](https://blog.darksky.net/)

**Comment by jordond** [comment]
Yeah :-1: very unfortunate.  I won't be adding anymore features, only fixing bugs, then archiving once the API is EOL.

**Comment by chrisdrackett** [comment]
let's all cross our fingers that apple plans on replacing this API with an apple one before the end of the line!

**Comment by jordond** [comment]
Looks like OpenWeatherMap has created a endpoint to match DarkSky, see [here](https://openweathermap.org/api/one-call-api)

**Comment by jordond** [comment]
The library is still a work in progress, but you can find it [here](https://github.com/worldturtlemedia/openweathermap-onecall). I have even mapped out the types for OpenWeatherMap vs DarkSky, and will be providing functionality to convert to the DarkSky format.

**Comment by jordond** [comment]
After a long hiatus... The new OneCall API wrapper is available!

See it [here](https://www.npmjs.com/package/owm-onecall-api)
